org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/compiler/XbaseCompiler.java

```diff
@@ -1397,7 +1398,11 @@ protected void _toJavaIfStatement(
 		}
 	}
 
-	private void closeBlock(ITreeAppendable caseAppendable) {
+	/** Close a block of code.
```

Comment:
Please follow the canonical style of writing JavaDoc. That would be a line break after the second star:
```
/**
 * Close the block of code.
 * ...
 */
```